var/spack/repos/builtin/packages/caliper/package.py

```diff
@@ -85,6 +85,7 @@ class Caliper(CMakePackage, CudaPackage, ROCmPackage):
     conflicts("+rocm+cuda")
 
     patch("for_aarch64.patch", when="target=aarch64:")
+    patch("sampler-service-missing-libunwind-include-dir.patch", when="@2.9.0 +rocm")
```

Comment:
Better `when="@2.9.0 +libunwind +sampler"`. The issue had nothing to do with rocm.

Reply:
Done! Thank you.